jsons/manifesto/it.json

```diff
@@ -0,0 +1,23 @@
+{
+  "title":"Manifesto per l'Artigianato del Software",
+  "raising_the_bar":"Alzare l'asticella.",
+  "introduction":"Come aspiranti Artigiani del Software, stiamo alzando l'asticella dello sviluppo professionale del software praticandolo e aiutando gli altri a imparare il mestiere.  Attraverso questo lavoro siamo arrivati a dare valore a:",
+  "not_only":"Non solo ",
+  "but_also":"ma anche ",
+  "working_software":"software funzionante,",
+  "well_crafted_software":"software ben fatto",
+  "responding_to_change":"rispondere al cambiamento,",
+  "steadily_adding_value":"valore aggiunto costante",
+  "individuals_and_interactions":"individui e interazioni,",
```

Comment:
I'd keep individui because even in the English language they could have chosen people instead of individuals. But the last word put the focus on the single person indeed as individual with his unique characteristics.

This is the definition of Treccani dictionary for individuo:
_Nel suo significato più generale, la parola INDIVIDUO indica ogni singolo ente in quanto distinto da altri della stessa specie..._


And this is for persona:
_Individuo della specie umana, senza distinzione di sesso, età, condizione sociale ecc., considerato sia come elemento a sé stante sia come facente parte di un gruppo o di una collettività..._

Reply:
Interesting explanation, I like the point made. I agree with you on keeping "individui".